### ISSUE TITLE
r1041_e82_400bps_sup_v430 model contains only 4 files

### ISSUE THREAD

**Issue by Axze-rgb** [HKU-BAL/Clair3#263]
Hello, according to the doc, the models should contain six files, but this one: r1041_e82_400bps_sup_v430; only contains:

```
ls
full_alignment.data-00000-of-00001  pileup.data-00000-of-00001
full_alignment.index                pileup.index

```
I downloaded it from Rerio. So far, Clair3 is running. Is it normal/expected? Do you think this will cause an issue? 

Thanks! 

Cheers
Alex

**Comment by zhengzhenxian** [comment]
Yes, it is expected, the updated tensorflow(v2.8.0) has merged two pre-trained data files into one single file. It would not affect variant calling performance.
